EnergyStar/EnergyStar.csproj

```diff
@@ -14,4 +14,14 @@
     <Folder Include="Config\" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="System.Text.Json" Version="6.0.5" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Update="settings.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
```

Comment:
Ideally, I would embed a default configuration in the executable so it can still run without extra configuration. User configuration can override (or add as an overlay) to default configuration if the file presents.

Reply:
That's a good idea. Actually we can generate a configuration if the file is missing, which is relatively easy to implement. But anyway, popping up an error dialog when settings file is missing is not user friendly.